gradio/external.py

```diff
@@ -398,7 +396,7 @@ def fn(*data):
             fns.append(fn)
         else:
             fns.append(None)
-    return gradio.Blocks.from_config(config, fns)
+    return gradio.Blocks.from_config(config, fns, iframe_url)
```

Comment:
When loading a private space, the inputs are shown correctly but the outputs are.

<img width="930" alt="image" src="https://user-images.githubusercontent.com/41651716/199767385-8a75640b-7670-415d-be77-042b10b63c94.png">

Testing with this private space: https://huggingface.co/spaces/gradio/test-loading-examples-private

Reply:
The boat image shows up on the left hand side but not in the dataset component. 

<img width="554" alt="image" src="https://user-images.githubusercontent.com/41651716/199769504-efd1e8bc-9076-4dd3-8eb2-f07efdffde97.png">

Reply:
Curious what behavior you see, I ran it again on the private space and now everything but the model3d shows up 🤔 

<img width="949" alt="image" src="https://user-images.githubusercontent.com/41651716/199770734-48da6346-417f-48a2-aee0-d45b40699040.png">

Reply:
Lol that's interesting. I get the same behavior as you the 2nd time -- everything but `Model3D` shows up. Nice catch, let me investigate

Reply:
This seems like a weird CORS issue and tbh I'm not sure what to do here. I think since this is a relatively niche use case (loading private Spaces, with (only?) the Model3D component), we can save for a future PR. I know @aliabid94 will be looking to CORS soon as well so flagging for your attention @aliabid94

Reply:
Sounds good with me. What makes you think it's a CORS issue?

Reply:
Will also not display the dataframe properly in the dataset if it's a private space:

<img width="1018" alt="image" src="https://user-images.githubusercontent.com/41651716/199813237-3defdebf-c125-41e6-be0e-044941d5080a.png">

Works fine for public spaces though. Confused as to why it would not impact images,audio, etc. Agree we can leave it for another issue but let's file it so we don't forget!